scripts/core/exp-command-buffer.yml

```
@@ -259,6 +259,84 @@ returns:
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Append a memory write command to a command-buffer object"
+class: $xCommandBuffer
+name: AppendMembufferWriteExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object."
+    - type: $x_mem_handle_t
+      name: hBuffer
+      desc: "[in] handle of the buffer object."
+    - type: "size_t"
+      name: offset
+      desc: "[in] offset in bytes in the buffer object."
+    - type: "size_t"
+      name: size
+      desc: "[in] size in bytes of data being written."
+    - type: "const void *"
```

the generator scripts don't handle a space before `*` well, I believe that's why the github actions failed.

```suggestion
    - type: "const void*"
```